q2_ms/types/_format.py

```diff
@@ -446,3 +447,29 @@ def _validate_(self, level):
 MatchedSpectraDirFmt = model.SingleFileDirectoryFormat(
     "MatchedSpectraDirFmt", "matched_spectra.txt", MatchedSpectraFormat
 )
+
+
+class MGFFileFormat(model.TextFileFormat):
+    def _validate(self):
+        try:
+            msms_data = mgf.read(str(self))
+
+            # Case where index exists but no spectra are found
+            if not isinstance(msms_data.get_by_index(0), dict):
+                raise ValidationError(
+                    "At least one spectrum must be present, but none were found."
+                )
+
+        except Exception:
+            raise ValidationError("Invalid MGF file.")
```

Comment:
```suggestion
        except Exception as e:
            raise ValidationError(f"Invalid MGF file. Original error: \n{e}")
```
Can you please add something like this. So the original error message can be seen.